persistence-cassandra-3.11/src/main/java/io/stargate/db/cassandra/impl/CassandraPersistence.java

```diff
@@ -285,6 +308,28 @@ public boolean isInSchemaAgreement() {
         <= 1;
   }
 
+  private boolean isStorageInSchemaAgreement() {
+    // See comment in isInSchemaAgreement()
```

Comment:
why not create a standard java-doc for this method(Instead of a simple comment)?
I was looking at the comment in the `isInSchemaAgreement()` and there is no mention of the `Stargate` node directly, only dead nodes: `and waiting for failed nodes is not a great idea.`
It also does not execute the `isGossipOnlyMember` method. Could you elaborate more?

Reply:
I've added javadoc and updated the _next_ comment line. I hope it clarified what is happening in this method, but please let me know if you think more info would be beneficial.

`isGossipOnlyMember` effectively means the endpoint is a Stargate node.